plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -734,8 +523,8 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
      */
     private ConnectorTableMetadata getTableMetadata(ConnectorSession session, SchemaTableName table)
     {
-        // getIcebergTable throws TableNotFoundException when table not found
-        org.apache.iceberg.Table icebergTable = getIcebergTable(session, table);
+        // catalog.loadTable throws TableNotFoundException when table not found
```

Comment:
Remove this comment and make it a Javadoc on the interface method. The comment isn't needed here, since the only alternative to throwing would be to return null (which we avoid and would need to be handled anyway).